src/extension/settingsHelper.ts

```diff
@@ -131,6 +132,20 @@ export class SettingsHelper {
         return "";
     }
 
+    public static getNetworkInspectorConsoleLogsColorTheme(): SystemColorTheme {
+        const workspaceConfiguration = vscode.workspace.getConfiguration(
+            "react-native-tools.networkInspector",
+            null,
+        );
+        if (workspaceConfiguration.has("consoleLogsColorTheme")) {
+            let consoleLogsColorTheme: string = ConfigurationReader.readString(
```

Comment:
It seems we may use `const` here
```suggestion
            const consoleLogsColorTheme: string = ConfigurationReader.readString(
```